code/modules/projectiles/ammo_datums.dm

```diff
@@ -1012,7 +1012,13 @@ datum/ammo/bullet/revolver/tp44
 	penetration = 80
 	sundering = 0
 
-/datum/ammo/bullet/sniper/incendiary
+/datum/ammo/bullet/sniper/amr
+	accurate_range_min = 7
```

Comment:
```suggestion
	accurate_range_min = 7
```
All these changes all in all seem like a severe overreaction and too sledgehammery, even for me considering all the changes in full the AMR has had.

Reply:
discussed in discord, accuracy a funni.